src/view/uniprotkb/components/UniProtCard.tsx

```diff
@@ -17,6 +22,45 @@ const UniProtCard: FC<{ data: UniProtkbAPIModel }> = ({
     (_): string => _.proteinDescription.recommendedName.fullName.value
   );
   const organismName = idx(data, (_): string => _.organism.scientificName);
+
+  const infoListData: { title: string; content: JSX.Element | string }[] = [];
+  if (data.genes) {
+    const convertedGeneNames = convertGeneNames(data.genes);
+    if (
+      convertedGeneNames.name ||
+      convertedGeneNames.alternativeNames.length > 0
+    ) {
+      infoListData.push({
+        title: 'Gene',
+        content: <GeneNamesView {...convertedGeneNames} />,
+      });
+    }
+  }
+
+  if (data.keywords) {
+    const categorisedKewywords = getKeywordsForCategories(data.keywords, [
```

Comment:
I noticed that the jira suggested "go with the order: function, biological process, disease, followed by the rest alphabetically?" Whereas the cards have only used the function, biological process, disease. Is that intentional?

Reply:
Yes, just checked with @SangyaP these are the ones selected by curators so we're trying with these first.